<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
  <head>
    <title>Dworkin Lab - Research</title>
    <meta http-equiv="Content-Style-Type" content="text/css" />
    <link type="text/css" rel="stylesheet" href="style.css" />
    <link rel="shortcut icon" href="favicon.ico" type="image/vnd.microsoft.icon" />
    <link rel="icon" href="favicon.ico" type="image/vnd.microsoft.icon" />
<!-- menu scroll -->
<!--[if lte IE 6]>
<style type="text/css">
/*<![CDATA[*/ 
html { overflow-x: auto; overflow-y: hidden; }
/*]]>*/
</style>
<![endif]-->
<!-- mouseovers -->
<script type="text/javascript">
<!--
if (window.document.images)  
{
// main menu
var research_i = new Image;
var research_o = new Image;
research_i.src = "images/buttons/research+.png";
research_o.src = "images/buttons/research.png";
var people_i = new Image;
var people_o = new Image;
people_i.src = "images/buttons/people+.png";
people_o.src = "images/buttons/people.png";
var publications_i = new Image;
var publications_o = new Image;
publications_i.src = "images/buttons/publications+.png";
publications_o.src = "images/buttons/publications.png";
var links_i = new Image;
var links_o = new Image;
links_i.src = "images/buttons/links+.png";
links_o.src = "images/buttons/links.png";
// side menu
var top_i = new Image;
var top_o = new Image;
top_i.src = "images/icons/arrow+.png";
top_o.src = "images/icons/arrow.png";
var home_i = new Image;
var home_o = new Image;
home_i.src = "images/icons/home+.png";
home_o.src = "images/icons/home.png";
}
// -->
</script>

  </head>
  <body>

<!-- side menu -->
<div id="menu" style="display:block; bottom:0px; right:0px; position:fixed;">
<table cellpadding="0" cellspacing="10px" border="0">
<tr><td><a href="#"
  onmouseover="if (window.document.images) {top.src=top_i.src}" 
  onmouseout="if (window.document.images) {top.src=top_o.src}"><img src="images/icons/arrow.png" alt="Top" border="0" name="top" id="top" /></a></td></tr>
<tr><td><a href="index.html"
  onmouseover="if (window.document.images) {home.src=home_i.src}" 
  onmouseout="if (window.document.images) {home.src=home_o.src}"><img src="images/icons/home.png" alt="Home" border="0" name="home" id="home" /></a></td></tr>
</table>
</div>

<!-- page content --><table width="800px" cellpadding="0" cellspacing="0" border="0" align="center">
<!-- titlebar --><tr><td width="800px" height="100px"><a href="index.html"><img src="images/titlebar.png" alt="Dworkin Lab" border="0" usemap="#map" /></a><map name="map" id="map"><area shape="rect" coords="406,15,415,24" href="photoshop/favicon.psd" alt="" /></map></td></tr>
<!-- horizontal rule --><tr><td width="800px" height="15px"><img src="images/hr.png" alt="" /></td></tr>
<!-- main menu --><tr><td width="800px" style="vertical-align: top; padding: 0px;"> 
<table width="800px" cellpadding="0" cellspacing="0" border="0" align="center"><tr>
<td align="center"><a href="research.html"
  onmouseover="if (window.document.images) {research.src=research_i.src}" 
  onmouseout="if (window.document.images) {research.src=research_o.src}">
  <img src="images/buttons/research.png" alt="Research" border="0" name="research" id="research" /></a></td>
<td align="center"><img src="images/buttons/dash.png" alt="" /></td>
<td align="center"><a href="people.html"
  onmouseover="if (window.document.images) {people.src=people_i.src}" 
  onmouseout="if (window.document.images) {people.src=people_o.src}">
  <img src="images/buttons/people.png" alt="People" border="0" name="people" id="people" /></a></td>
<td align="center"><img src="images/buttons/dash.png" alt="" /></td>
<td align="center"><a href="publications.html"
  onmouseover="if (window.document.images) {publications.src=publications_i.src}" 
  onmouseout="if (window.document.images) {publications.src=publications_o.src}">
  <img src="images/buttons/publications.png" alt="Publications" border="0" name="publications" id="publications" /></a></td>
<td align="center"><img src="images/buttons/dash.png" alt="" /></td>
<td align="center"><a href="links.html"
  onmouseover="if (window.document.images) {links.src=links_i.src}" 
  onmouseout="if (window.document.images) {links.src=links_o.src}">
  <img src="images/buttons/links.png" alt="Links" border="0" name="links" id="links" /></a></td>
</tr></table>
<!-- /main menu --></td></tr>
<!-- horizontal rule --><tr><td width="800px" height="15px"><img src="images/hr.png" alt="" /></td></tr>


<!-- content --><tr><td width="800px" style="vertical-align: top; padding: 10px;"> 
<span class="header">Scientific Interests</span></a><br />
<br />
  Our research focuses on using genetic and genomic approaches to address outstanding questions in the evolution of complex phenotypes. We primarily use <i>Drosophila</i> (among other systems) as a model system to examine the interplay of standing genetic variation and environmental heterogeneity and their contribution to trait expression and variation; and how this interaction is itself shaped by evolutionary forces such as natural selection and drift. While some research programs focus on either the genetics of trait expression, traditionally using mutations (or other perturbations) of large effect, or on genetic variation within populations, utilizing natural variation (usually with much more subtle phenotypic effects), we successfully integrate both of these approaches along with genomic methods (transcriptional profiling, genome wide mapping) into our work to answer our questions. Details about our research can be found below.<br />
<br />
<a href="#wing">Evolutionary genetics and genomics of wing shape in <i>Drosophila</i><br />
<a href="#genback">Genetic and genomic approaches for the study of conditional effects of mutations. </a><br />
<a href="#crypt">Cryptic Genetic Variation in natural populations</a><br />
<a href="#mant">Evolution of anti-predation traits: Experimental Evolution using <i>Drosophila</i></a><br />
<a href="#col">Collaborative research on the evolution of scaling relationships (in numerous organisms)</a><br />
<br />



<!-- /content --></td></tr>
<!-- horizontal rule --><tr><td width="800px" height="15px"><img src="images/hr.png" alt="" /></td></tr>
<!-- content --><tr><td width="800px" style="vertical-align: top; padding: 10px;">
<a nohref name="wing"></a><span class="header">Dissecting the genetic architecture of wing shape in <i>Drosophila</i></span><br />
<br />
One of the main thrusts of our lab involves the genetic analysis of wing shape in <i>Drosophila</i>. One of the main goals is to elucidate the genetic architecture of complex phenotypes where there is a complex interplay between genetic and environmental influences generating the phenotypic variation observed. Wing shape represents a complex multivariate phenotype, yet is sufficiently straightforward to measure that we can perform large manipulative experiments, in a genetically tractable organism. To assess variation for wing shape we utilize a combination of induced mutations, controlled environmental variation, natural genetic variation within species and inter-specific differences between closely related species.<br />
<br />
To elucidate the proximate mechanisms that shape trait expression, we utilize induced mutations in highly controlled genetic backgrounds, as this allows us to test the effects of individual genetic perturbations (<a href="publications/"Dworkin Gibson 2006.pdf">Dworkin and Gibson 2006</a>).  In addition we also examine how these genetic effects interact with the environment to influence shape, work done in conjunction with Vincent Debat (<a href="publications/Plasticity2009.pdf">Debat et. al. 2009</a>). We have examined the relationship between subtle genetic perturbations on wing shape and how it co-varies with mRNA transcript abundance assayed on a whole genome scale (<a href="publications/DworkinEtAlGenetics2011.pdf">Dworkin et. al. 2011</a>). Further to this goal, work from two undergraduates in the lab (Alycia Kowalski and Lindy Johnson) examined the interaction between these subtle 
genetic perturbations to assess whether their joint effects are consistent with predicted effects based on developmental genetic network topologies. 
We have been developing new statistical tools to help analyze such data, and in particular to avoid the problem of reducing a 
complex high dimensional space to a low dimensional space (i.e. the problem with dimensional reduction techniques such as principal components analysis, or canonical variates).<br />
<br />
<img src="images/research/spi_series.png" alt="spi" />
An allelic series of <i>spitz</i>.
<br />
<br />
In many ways the work with the induced mutations provides the fodder for our primary interest;  what explains variation for wing shape in natural populations? Wing size and shape varies considerably in <i>Drosophila</i>, in particular with respect to latitude, and as we really described, for altitude as well (<a href="publications/PitchersPoolDworkin2013.pdf">Pitchers et. al. 2013</a>). Initial work examined the associations between polymorphisms in candidate wing genes with variation in shape in large natural populations (<a href="">Dworkin et al. 2005</a>), where we are finishing a large study focusing on functionally characterized cis-regulatory modules (<a href="">Pitchers et al in Prep</a>). We have completed the initial association using a panel of inbred lines, and are generating transgenic reporter constructs for haplotypes that may be associated with wing shape and appear to be under selection. <br />
<br />
In collaboration with David Houle (Florida State University) we are now utilizing a panel of strains (derived from natural populations) of Drosophila (DGRP panel) to perform genome wide association mapping (GWAS) for wing shape, which will aid in identifying common polymorphisms of moderate effect that contribute to wing shape. These genetic associations will then be replicated in a large cohort of wild-caught flies to assess their genetic contributions in the wild. This is part of a large, ambitious project to generate a predictive evolutionary model to determine whether we have sufficient knowledge to predict the response in selection (with respect to both phenotypic and genetic effects). We will test these predictions by performing artificial selection along a number of shape trajectories.
<br />
<br />
<img src="images/research/variate2.png" alt="var1" /><br />
Epistatic interaction between mutations and its effect on shape.
<br />


<!-- /content --></td></tr>
<!-- horizontal rule --><tr><td width="800px" height="15px"><img src="images/hr.png" alt="" /></td></tr>
<!-- content --><tr><td width="800px" style="vertical-align: top; padding: 10px;"> 
<a nohref name="genback"></a><span class="header">Genetic and genomic approaches to the analyses of conditional effects of mutations. </span><br />
<br />
It is well known that the phenotypic effects of any given mutation depends on both the genetic background in which it is observed, and on the environment in which the organism develops and lives in (<a href="publications/ChandlerChariDworkin2013_TIG.pdf">Chandler et al. 2013</a>). While quantitative genetics has traditionally studied the marginal effects of these alleles (averaged across genotypes, conditional upon the environment), it is clear that context matters a great deal more than such a simple approach has traditionally allowed. We are utilizing functionally characterized, induced mutations to study the impact of genetic background and rearing environment on the phenotypic expression of these mutations (quantitative variation in mutant expressivity). Our work goes beyond the basic characterization of genetic background to systematically address higher order genetic effects (beyond the marginal effects of genetic background on a mutant) to characterize the effects of 
genetic background and environmental effects on the ordering of mutational effects (allelic series), intra-genic (complementation) 
and inter-genic (epistasis) interactions across a number of genes and phenotypes including developmental, morphological, behavioral as 
well as the transcriptome. We have already demonstrated (<a href="publications/DworkinKennerlyTackHutchisonBrownMahaffeyGibson.pdf">Dworkin et al. 2009</a>) that the epistatic interactions between mutations 
in the scalloped and bifid genes are background dependent, with the effects in one background showing functional epistasis 
(one allele suppressing the effects of the other), while in the other background the effects are synergistic 
(greater than additive). Work in the lab continues to address this on a genome wide scale, 
with recent work demonstrating that the vast majority of genetic interactions among induced mutations being 
background dependent (<a href="http://www.plosgenetics.org/doi/pgen.1003661 ">Chari & Dworkin In Press</a>. also see <a href="http://sciencenews.org/view/generic/id/58272/title/Mutation_effects_often_depend_on_genetic_milieu">Science News</a>). 
We are currently investigating these effects across multiple environments, and utilizing transcriptional profiling to assess the genomic consequences of the conditional effects of these mutations.<br />
<br />
<img src="images/scallopped.png" alt="Scallopped" /><br />

<br />

<!-- /content --></td></tr>
<!-- horizontal rule --><tr><td width="800px" height="15px"><img src="images/hr.png" alt="" /></td></tr>
<!-- content --><tr><td width="800px" style="vertical-align: top; padding: 10px;"> 
<a nohref name="crypt"></a><span class="header">Cryptic Genetic Variation in natural populations</span><br />
<br />
Complementary to our functional genetic studies of the conditional effects of mutations, we are also examining the flip-side, namely genetic variation that is only conditionally expressed depending on environmental (GxE) or genetic (GxG) context.  While such cryptic genetic variation  occurs in natural populations (<a href="">Dworkin et al. 2003</a>; <a href="">Dworkin et al. 2005</a>) , what is unclear is the fraction of standing genetic variation that is phenotypically hidden and thus unavailable to natural selection, and whether such variation is important for adaptive evolution. One important question is whether such variation is truly cryptic at all, or has subtle pleiotropic effects.  To address this we are currently mapping the background modifiers for the mutations discussed above to determine what (if any) phenotypic and fitness consequences they may have in natural populations. 
In addition Sudarshan Chari in the lab is exploring the potential role of standing genetic variation in populations for 1) 
providing compensatory effects to new (via mutation or introgression) deleterious alleles, and 2) Understanding how natural and 
artificial selection results in a re-wiring of the developmental genetic network for the target (optimal) phenotypes, after network perturbation. Sudarshan has discussed some of this work on a recent <a href="http://beacon-center.org/blog/2013/07/02/beacon-researchers-at-work-visualizing-and-understanding-context-dependence-in-evolution/"> blog post.</a>  Chris Chandler, a post-doctoral 
researcher will also be studying questions relating to compensatory mutations within the context of experimental manipulations for the opportunity for sexual selection.
<br />






<!-- /content --></td></tr>
<!-- horizontal rule --><tr><td width="800px" height="15px"><img src="images/hr.png" alt="" /></td></tr>
<!-- content --><tr><td width="800px" style="vertical-align: top; padding: 10px;"> 
<a nohref name="mant"></a><span class="header">Evolution of anti-predation traits using experimental evolution. </span><br />
<br />
The newest and perhaps most exciting research program that has begun in the lab uses Drosophila as a model system to study the evolution of anti-predation traits, both morphological and behavioral. We have initiated a long-term experimental evolution study of <i>Drosophila</i> under the pressure of predation by juvenile preying mantids (<i>Tenodera aridifolia sinensis</i>) and by the zebra jumping spider (<i>Salticus scenicus</i>). Our work has demonstrated phenotypic selection on both wing size and shape due to predation, and these have changed in a complex manner in the evolved populations. We are continuing this work by exploring how the multivariate selection gradient has changed after 30 generations of selection. Michael DeNieu, a graduate student in the lab has been spearheading the experimental evolution project, and is currently examining a suite of behaviors in the evolved lines, and is examining these and other strains for potential trade-offs. Read (and see a video) about Michael's work with <i>Drosophila</i> and one of its predators <a href="http://beacon-center.org/blog/2011/12/05/beacon-researchers-at-work-experimenting-with-predation/">here</a>. Abhijna Parigi who has recently joined the lab continues a great deal of behavioural work on this front.<br />
<br />
<table width="100%" cellpadding="0" cellspacing="0px" border="0" align="center">
  <tr>
    <td width="400px"><img src="images/research/MantisNom.jpg"/></td>
    <td width="400px"><img src="images/research/Spidernom.png"/></td>
  </tr>
</table>
<br />
A juvenile mantis (left) and a jumping spider (right) eating <i>Drosophila</i>.
<br />
<br />
<table width="100%" cellpadding="0" cellspacing="0px" border="0" align="center">
  <tr>
    <td width="410px"><img src="images/research/pred3.png" alt="feast" /></td>
    <td>Variation for wing size may contribute to predator evasion.<td>
  </tr>
</table>




<!-- /content --></td></tr>
<!-- horizontal rule --><tr><td width="800px" height="15px"><img src="images/hr.png" alt="" /></td></tr>
<!-- content --><tr><td width="800px" style="vertical-align: top; padding: 10px;"> 
<a nohref name="col"></a><span class="header">Collaborative research on the evolution of scaling relationships.</span><br />
<br />
In addition to the main thrusts of research that occur in my lab, I am actively involved with numerous collaborations, here at Michigan State University, across the country and internationally. What integrates <B>all</B> of these collaborative projects are questions pertaining to the proximate and ultimate mechanisms involved with the evolution of scaling relationships within organisms. <br />
<br />
In conjunction with <a href="https://www.msu.edu/~shingle9/">Alex Shingleton</a> here at Michigan State University, and <a href="http://bchs.uh.edu/~frankino/">Tony Frankino</a> at the University of Houston, we are examining some of the proximate genetic mechanisms that contribute to natural variation for scaling relationships in <i>Drosophila melanogaster</i>. We are utilizing artificial selection to alter scaling relationships (wing vs. body size) using a large natural population. We will examine the correlated responses of selection with regard to physiological & developmental processes (Shingleton), functional responses (Frankino & Dworkin) and to map the underlying genetic regions contributing to the selection response (Dworkin).  In addition Alex Shingleton and I have been jointly examining the consequences of how different environmental gradients such as rearing temperature, 
density and nutrition, alter patterns of multivariate allometry (<a href="publications/2625.full.pdf">Shingleton et al. 2009</a>).<br />
<br />
In conjunction with <a href="http://www.kbs.msu.edu/research/faculty/conner">Jeff Conner</a> and <a href="http://shiulab.plantbiology.msu.edu/wiki/index.php/Main_Page">Shin-Han Shiu</a> in Plant Biology we are examining the evolution of the anther exsertion phenotype in the wild radish <i>Raphanus raphanistrum</i>. The anther exsertion phenotype represents the scaling relationship between anther and corolla height, and has been shown by Jeff Conner to be highly conserved in the <i>Brassicaceae</i>, and is under natural selection in radish. Using Jeff's  lines artificially selected divergently on this phenotype, we are in the midst of generating genomic and genetic tools for wild radish to help to understanding the underlying genetic mechanisms that contribute to variation for this phenotype. This fantastic system allows us to integrate the substantial ecological and evolutionary work 
(in particular measures of phenotypic selection and the role of pollinators) with the genetics to get a far more complete picture of the evolution of this phenotype.<br />
<br />
For more information on this project please visit the <a href="http://radish.plantbiology.msu.edu/index.php/Main_Page">radish database.</a> <br />
<br/>

In conjunction with a graduate student Eli Swanson, I have become involved with a study on the evolution of multivariate allometry, and its relationship with lifetime reproductive success. Eli is a current graduate student of <a href="https://www.msu.edu/~holekamp/">Kay Holekamp</a>, with an interest in multivariate evolution, and has been utilizing the long term data collected by Kay to test whether different allometric components of size (i.e. those traits that scale to a greater or lesser extent with idealized size) may contribute to measures of fitness. <br />
<br />
In addition to the collaborative projects here at Michigan State University, I am also involved with a project with <a href="http://dbs.umt.edu/research_labs/emlenlab/personal.htm">Douglas Emlen</a> and <a href="http://entomology.wsu.edu/personal/laura_lavine/">Laura Lavine</a> exploring the evolution of scaling relationships in dung, scarab and rhinoceros beetles, each of which have highly exaggerated phenotypes such as horns, or in the case of stag beetles the mandibles. In each of these groups individuals may be polyphenic for these traits based on both size and/or sex, allowing us to utilize inherent natural variation in the scaling relationships. Utilizing one species from each of these clades we are generating genomic resources to assess the transcriptional profiles of tissues leading to both exaggerated (hyper-allometric) phenotypes, as well as those that are scale invariant.  We believe this will aid in our understanding of the genetic basis of the evolution of exaggerated traits.
<br />
<br />






<!-- /content --></td></tr>
<!-- end page content --></table>

  </body>
</html>
